From c04253e27be667681644e653ea24677c2ad90880 Mon Sep 17 00:00:00 2001
From: Awais Belal <awais_belal@mentor.com>
Date: Wed, 16 Nov 2016 15:11:33 +0500
Subject: [PATCH] WebServer: fix misleading cleanup under RemoveHandlers

The RemoveHandlers implementation had a terminating ';'
which would not allow the cleanup of the whole signal
map.

Signed-off-by: Awais Belal <awais_belal@mentor.com>
---
 CodeXL/Components/Graphics/Server/WebServer/OSDependent.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CodeXL/Components/Graphics/Server/WebServer/OSDependent.cpp b/CodeXL/Components/Graphics/Server/WebServer/OSDependent.cpp
index ef92962..e344e7a 100755
--- a/CodeXL/Components/Graphics/Server/WebServer/OSDependent.cpp
+++ b/CodeXL/Components/Graphics/Server/WebServer/OSDependent.cpp
@@ -1089,7 +1089,7 @@ public:
     void RemoveHandlers()
     {
         std::map<int, SignalData>::iterator it;
-        for (it = m_signalMap.begin(); it != m_signalMap.end(); ++it);
+        for (it = m_signalMap.begin(); it != m_signalMap.end(); ++it)
         {
             sigaction(it->first, &(it->second.old_action), nullptr);
         }
-- 
1.9.1

